.github/workflows/test-documentation.yml

```diff
@@ -0,0 +1,80 @@
+name: Test Documentation
+
+# run CI on pushes to master, and on all PRs (even the ones that target other
+# branches)
+
+on:
+ push:
+   branches: [master]
+ pull_request:
+
+jobs:
+  documentation:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ['14.x']
+    steps:
+
+    - name: Checkout agoric-sdk
+      uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: cache node modules
+      uses: actions/cache@v1
+      with:
+        path: ~/.cache/yarn
+        key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+
+    # Select a branch on documentation to test against by adding text to the body of the
+    # pull request. For example: #documentation-branch: zoe-release-0.7.0
+    # The default is 'master'
+    - name: Get the appropriate documentation branch
+      id: get-branch
+      uses: actions/github-script@0.9.0
+      with:
+        result-encoding: string
+        script: |
+          let branch = 'master';
+          if (context.payload.pull_request) {
+            const { body } = context.payload.pull_request;
+            const regex = /.*\#documentation-branch:\s+(\S+)/;
+            const result = regex.exec(body);
+            if (result) {
+              branch = result[1];
+            }
+          }
+          console.log(branch);
```

Comment:
What do you think of logging only after line 49 with a message indicating that the branch has been overridden?

Reply:
Yeah, that could work. There have been times when it was nice to see "master" to ensure the process is working at all. I don't think the logging is costing us anything since it's logging to CI output